typescript-ignorant/Java/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+java -jar "$HOME/opt/antlr-4.7.2-complete.jar" ../TypeScriptLexer.g4 ../TypeScriptParser.g4
+mv ../*.java ./
+
+CLASSPATH=".:$HOME/opt/antlr-4.7.2-complete.jar:$CLASSPATH" javac *.java
+
```

Comment:
Why removing this script? It will make work easier for those who will improve the grammar after me (if anybody will).

Reply:
Different users use different tools for grammar checking. For instance, CI uses maven plugin for grammar checking. We are trying to avoid helper files in the official repository.